packages/core/database/lib/entity-manager/regular-relations.js

```diff
@@ -242,21 +241,27 @@ const cleanOrderColumns = async ({ id, attribute, db, inverseRelIds, transaction
         )
         .transacting(trx);
       break;
-    default:
-      await db
-        .getConnection()
+    default: {
+      const schemaName = db.connection.getSchemaName();
+      let joinTableName = joinTable.name;
+      if (schemaName) {
+        joinTableName = `${schemaName}.${joinTableName}`;
+      }
+
+      await db.connection
         .raw(
           `UPDATE ?? as a
-            SET ${update.join(', ')}
-            FROM (
-              SELECT ${select.join(', ')}
-              FROM ??
-              WHERE ${where.join(' OR ')}
-            ) AS b
-            WHERE b.id = a.id`,
-          [joinTable.name, ...updateBinding, ...selectBinding, joinTable.name, ...whereBinding]
+              SET ${update.join(', ')}
+              FROM (
+                SELECT ${select.join(', ')}
+                FROM ??
+                WHERE ${where.join(' OR ')}
+              ) AS b
+              WHERE b.id = a.id`,
+          [joinTableName, ...updateBinding, ...selectBinding, joinTableName, ...whereBinding]
```

Comment:
I don't think it works. I think you need to do `??.??` and `[schemaName, joinTableName, ...]` as each argument is either an identifier or a value. `${schemaName}.${joinTableName}` would be 2 identifiers at the same time. Did you try it?

I think you can either: 
- directly put the value in the query, not using the arguments
- don't handle the schema for now, we will do it in another PR anyway as there are other parts of the app that needs to be fixed with the schema

I would suggest the second options so we can also take the time to do it (not doing it during the rush)

Reply:
Apparently that works indeed! https://knexjs.org/guide/ref.html#ref
![Capture d’écran 2022-11-16 à 11 56 05](https://user-images.githubusercontent.com/11708220/202162393-844ed624-1ee8-4877-9dc4-769202c23791.png)

Good to know. However I would still not do that in that PR for the moment as it would be add some inconsistency